include/rocksdb/utilities/object_registry.h

```diff
@@ -38,6 +39,23 @@ using FactoryFunc =
 // library
 using RegistrarFunc = std::function<int(ObjectLibrary&, const std::string&)>;
 
+using PluginFunc = std::function<int(Plugin*, size_t, std::string*)>;
```

Comment:
What are the parameters and return value? Note Google C++ guidelines "put all input-only parameters before any output parameters."

Reply:
Added documentation